test/cleanup.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// cleanup allows you to define a cleanup function that will be executed
+// if your test is interrupted.
+
+package test
+
+import (
+	"os"
+	"os/signal"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CleanupOnInterrupt will execute the function cleanup if an interrupt signal is caught
+func CleanupOnInterrupt(cleanup func()) {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
+	go func() {
+		for range c {
+			cleanup()
+			os.Exit(1)
+		}
+	}()
+}
+
+// TearDown will delete created names using clients.
+func TearDown(clients *Clients, names ResourceNames) {
+	if clients != nil && clients.KnativeServingAlphaClient != nil {
+		clients.KnativeServingAlphaClient.KnativeServings.Delete(names.KnativeServing, &metav1.DeleteOptions{})
+	}
+
+	// Remove all the deployments
+	dpListSave, err := clients.KubeClient.Kube.AppsV1().Deployments(names.Namespace).List(metav1.ListOptions{})
```

Comment:
This isn't necessary. Just deleting the KS instance should suffice

Reply:
OK.